jcommune-view/jcommune-web-view/src/main/webapp/resources/javascript/app/userSearch.js

```diff
@@ -38,8 +38,8 @@ userSearch.connectionErrorCallback = function(xhr, status) {
         } else {
             userSearch.popup.show($labelConnectionLostGenericError);
         }
-    } else if (xhr.status == 403) {
-        userSearch.popup.show($labelNotLoggedInError);
+    } else if (xhr.status == 403 || xhr.status == 500 || xhr.status == 302) {
+        userSearch.popup.show($labelNotLoggedInGenericError);
```

Comment:
The errors don't seem to match the statuses.. 403 is access denied - agree. But error 500 is not. Neither 302 (though I assume this is a redirect to the login page?) - this we can change back in the security config if you think it'd be better.